contrib/yggdrasil-brute-simple/LICENSE

```diff
@@ -0,0 +1 @@
+As public domain as possible. Don't blame me if your computer explodes.
```

Comment:
I think CC0 might be more legal way to say this. https://creativecommons.org/choose/zero/

Reply:
Apparently CC0 has some problems with explicitly not waiving patent rights.
https://opensource.stackexchange.com/questions/133/how-could-using-code-released-under-cc0-infringe-on-the-authors-patents

I'll probably use the Unlicense as it seems to be fairly short boilerplate for releasing code into the public domain and disclaiming warranty.

I expect to rarely (hopefully never) have to modify this code and I would rather not complicate the process of `git clone`-ing.